cmd/gateway/zcn/gateway-zcn.go

```diff
@@ -549,13 +549,8 @@ func (zob *zcnObjects) PutMultipleObjects(
 	}
 	operationRequests := make([]sdk.OperationRequest, total)
 	objectInfo := make([]minio.ObjectInfo, total)
-	cb := &statusCB{
-		doneCh: make(chan struct{}, 1),
-		errCh:  make(chan error, 1),
-	}
-
 	var wg sync.WaitGroup
-	errCh := make(chan error, total)
+	errCh := make(chan error)
```

Comment:
you can't remove size, since goroutine will hang on writ and never call wg.done(), main routine will never call channel close in that case